packages/transcription/transcribe/oauth.py

```diff
@@ -0,0 +1,21 @@
+
+import os
+from dotenv import load_dotenv
+
+def import_env_vars():
+    os.chdir(r"packages\transcription")
+    load_dotenv(r"cred\cred.env")
+
+    # Get credentials from environment variables
+    YOUTUBE_API_KEY = os.getenv("YOUTUBE_API_KEY")
+    CLIENT_ID = os.getenv("CLIENT_ID")
```

Comment:
Where do CLIENT_ID and CLIENT_SECRET come from?

Reply:
@mikafur32 is the one who built this - i believe a config file but i could be wrong